client/src/app/tabs/bpmn/modeler/features/save-editing/SaveEditing.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+export default function SaveEditing(editorActions, injector) {
+  editorActions.register('saveEditing', function() {
+    var directEditing = injector.get('directEditing', false);
```

Comment:
Can't we assume, that `directEditing` is available in all Editors which will use this module? Therefore we could inject it directly without the additional `injector` dependency.

Reply:
The directEditing dependency is not available for DMN Table so we need to check if it exists here. If we don't register this action for DMN Tables, then we get the NotRegistered exception. So it needs to be registered but do nothing.

Reply:
The other solution would be to implement another class only for DMN Tables which registers saveEditing to noop function, but I find the current solution better.